thunder/tests/distributed/test_fsdp.py

```diff
@@ -28,6 +28,8 @@
     TE_AVAILABLE,
 )
 
+from thunder.executors.transformer_engine_v2ex import transformer_engine_v2_ex, TransformerEngineTransformV2
```

Comment:
Similar to above comment - In a follow-up PR, I think we should create a new file for TE distributed tests